dev-libs/wlc/wlc-9999.ebuild

```diff
@@ -29,7 +29,10 @@ RDEPEND="virtual/opengl
 		 x11-libs/xcb-util-image
 		 x11-libs/xcb-util-wm
 		 x11-libs/libXfixes )
-	xwayland? ( x11-base/xorg-server[wayland] )
+	xwayland? ( x11-libs/libxcb[xkb]
```

Comment:
multi-dep conditionals should always be done like
```
xwayland? (
	x11-libs/libxcb[xkb]
	x11-libs/xcb-util-image
	x11-libs/xcb-util-wm
	x11-base/xorg-server[wayland]
)
```
as that makes it easier for the eye to find the end